social_protection/schema.py

```diff
@@ -0,0 +1,47 @@
+import graphene
+from django.contrib.auth.models import AnonymousUser
+from django.db.models import Q
+
+from core.schema import OrderedDjangoFilterConnectionField
+from core.utils import append_validity_filter
+from social_protection.apps import SocialProtectionConfig
+from social_protection.gql_mutations import CreateBenefitPlanMutation, UpdateBenefitPlanMutation, \
+    DeleteBenefitPlanMutation
+from social_protection.gql_queries import BenefitPlanGQLType
+from social_protection.models import BenefitPlan
+import graphene_django_optimizer as gql_optimizer
+
+
+class Query:
+    benefit_plan = OrderedDjangoFilterConnectionField(
+        BenefitPlanGQLType,
+        orderBy=graphene.List(of_type=graphene.String),
+        dateValidFrom__Gte=graphene.DateTime(),
+        dateValidTo__Lte=graphene.DateTime(),
+        applyDefaultValidityFilter=graphene.Boolean(),
+        client_mutation_id=graphene.String()
+    )
+
+    def resolve_benefit_plan(self, info, **kwargs):
+        filters = []
+        filters += append_validity_filter(**kwargs)
```

Comment:
This should work as well
```suggestion
        filters = append_validity_filter(**kwargs)
```

Reply:
done